shared/src/main/scala/amf/core/client/common/render/JSONSchemaVersion.scala

```diff
@@ -0,0 +1,25 @@
+package amf.core.client.common.render
+
+import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
+
+@JSExportAll
+@JSExportTopLevel("JSONSchemaVersions")
+object JSONSchemaVersions {
+  import amf.core.client.common.render.{Unspecified => UnspecifiedObj}
+  val Unspecified: JSONSchemaVersion = UnspecifiedObj
+  val Draft04: JSONSchemaVersion     = JsonSchemaDraft4
+  val Draft07: JSONSchemaVersion     = JsonSchemaDraft7
+  val Draft201909: JSONSchemaVersion = JsonSchemaDraft201909
+}
+
+@JSExportAll
+sealed trait JSONSchemaVersion
+
+@JSExportAll
+object Unspecified extends JSONSchemaVersion
+@JSExportAll
+object JsonSchemaDraft4 extends JSONSchemaVersion
```

Comment:
Change to case object with a string so that it can be used from JS

Reply:
followed the same mechanism as ValidationMode, don't understand why it cant be used in JS.